docs/notebooks/Poincare Evaluation.ipynb

```diff
@@ -1318,18 +1218,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 149,
+   "execution_count": 56,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/html": [
        "<table>\n",
-       "    <tr>\n",
-       "        <th colspan=\"1\" style=\"text-align:left\">WordNet Link Prediction</th>\n",
-       "        <th colspan=\"1\"></th>\n",
-       "        <th colspan=\"6\" style=\"text-align:center\"> Dimensions</th>\n",
-       "    </tr>\n",
+       "        <tr>\n",
```

Comment:
Why diff here?

Reply:
This wasn't done manually, when i ran the notebook in the docker container the output might have been rendered differently.